404.html

```diff
@@ -7,3 +7,8 @@
 
 <h1 id="error-message"></h1>
 <ul id="link-list"></ul>
+<hr>
+<br>
+<br>
+<h4>Something wrong? Don't think you should have ended up here?</h4>
+<p>Send us an email at <a href=/email>cssoc@cs.bham.ac.uk</a> or reach out to one of <a href=/committee>committee</a>.</p>
```

Comment:
*the* committee

Reply:
Also, could you add an option for "contact one of the @committee on (the?) (CSS?) Discord"

Reply:
congrats on @-ing someone lmao

Reply:
oops

Reply:
I can add "the" easily, but there's no clean way to add a reference to @\Committee via discord easily.

Reply:
unless you want the @\Committee reference on the /committee page

Reply:
I don't mean to actually link to committee on discord, I mean literally just put the @ there to indicate that it's a discord role. A normal link to the discord can be added to the word "Discord" too